nesc/whip6/platforms/boards/cc26xxbased/PutcharProviderPub.nc

```diff
@@ -25,9 +28,9 @@ implementation {
     // -- no putchar
     components DummyPutcharProviderPub;
 #else
-    // -- putchar over UART0
```

Comment:
This should not be deleted but rather ifdefed. We may yet want to have a putchar provider over something else than the sensor controller.